modules/ROOT/pages/_partials/sync/data-sync-technical-overview.adoc

```diff
@@ -0,0 +1,27 @@
+[id="data-sync-technical-overview-{context}"]
+= Data Sync Technical Overview
+
+This section describes the technical aspects of Data Sync.
+
+// tag::excludeDownstream[]
+
+image::data-sync-technical-overview.png[Data Sync Technical Overview]
+
+// end::excludeDownstream[]
+
+// tag::excludeUpstream[]
+
+image::data-sync-technical-overview.png[Data Sync Technical Overview]
+
+// end::excludeUpstream[]
+
+.Data Sync case study
```

Comment:
remove this case study too

Reply:
Completed as part of https://github.com/aerogear/mobile-docs/commit/48a7e8cda0cbedc0b2f7e5bc5e91730af1cb6899